src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -91,10 +101,22 @@ def load(self) -> list[Rule]:
             msg = "Please run: databricks labs ucx table-mapping"
             raise ValueError(msg) from None
 
-    def skip_table(self, backend: StatementExecutionBackend, schema: str, table: str):
+    def get_tables_to_migrate(self, tables_crawler: TablesCrawler):
+        rules = self.load()
+        crawled_tables = tables_crawler.snapshot()
+
+        # Getting all the source tables from the rules
+        source_databases = {rule.src_schema for rule in rules}
+        databases_in_scope = self._get_databases_in_scope(source_databases)
+
+        return self._get_tables_in_scope(rules, databases_in_scope, crawled_tables)
```

Comment:
```suggestion
        return self._get_tables_in_scope(tables_crawler)
```

Reply:
It makes rules harder to test in testing.

Reply:
you neven test private methods. if you do - you're doing terribly wrong.

Reply:
Addressed